Lesson13/Lesson13.cpp

```diff
@@ -0,0 +1,295 @@
+#include <iostream>
+#include <algorithm>
+#include <chrono>
+#include <fstream>
+#include <vector>
+#include <string>
+
+// small vacabulary from https://copylists.com/words/list-of-5-letter-words/
+// middle vacabulary from https://github.com/charlesreid1/five-letter-words/blob/master/sgb-words.txt
+// need encript the save file to prevent hacking/cheeting guessing words
+// need store save file on a server or at least dobling time variable in windows's registry to prevent time cheeting
+// program works correctly ONLY on WINDIWS machine
+
+static const bool QA_MODE = true; // shows hidden tips :)
+static const unsigned long long secondsMinutesHours = 86400;
+static const unsigned long long nanoseconds = 10000000;
+static const std::string saveFileName = "save.ini";
+static const std::string dictionaryFileName = "words.txt";
+static unsigned long long timeForDailyGame = 0;
+
+enum GameState {
+    INIT = 1,
+    MENU,
+    RANDOM_GAME_MODE,
+    DAILY_GAME_MODE,
+    EXIT
+};
+
+void caesarsEncript() {
+    // enctipt the word before save it into readable file
+}
+
+void caesarsDecript() {
+    // decript the word after reading from visually unsafe file
+}
+
+void loadDictionaryFromFile(std::vector<std::string>& dictionary) {
+    std::ifstream iFile(dictionaryFileName);
+
+
+    if (iFile.is_open()) { // read dictionary file
+        std::string line;
+
+        while (getline(iFile, line)) {
+            std::transform(line.begin(), line.end(), line.begin(), ::toupper);
+            dictionary.push_back(line);
+        }
+        iFile.close();
+    }
+}
+
+bool tryToLoadSavedGamesFromFile() {
```

Comment:
just opening and closing file? That's strange and inefficient

Reply:
agree, i was in a hurry so i just tried to solve "no existing file" problem as fast as first thought. 😅